pkg/endpoint/endpoint.go

```diff
@@ -174,32 +174,34 @@ const (
 // Endpoint contains all the details for a particular LXC and the host interface to where
 // is connected to.
 type Endpoint struct {
-	ID               uint16       // Endpoint ID.
-	Mutex            sync.RWMutex // Protects all variables from this structure below this line
-	DockerID         string       // Docker ID.
-	DockerNetworkID  string       // Docker network ID.
-	DockerEndpointID string       // Docker endpoint ID.
-	IfName           string       // Container's interface name.
-	LabelsHash       string
-	OpLabels         pkgLabels.OpLabels
-	LXCMAC           mac.MAC               // Container MAC address.
-	IPv6             addressing.CiliumIPv6 // Container IPv6 address.
-	IPv4             addressing.CiliumIPv4 // Container IPv4 address.
-	IfIndex          int                   // Host's interface index.
-	NodeMAC          mac.MAC               // Node MAC address.
-	NodeIP           net.IP                // Node IPv6 address.
-	SecLabel         *policy.Identity      // Security Label  set to this endpoint.
-	PortMap          []PortMap             // Port mapping used for this endpoint.
-	Consumable       *policy.Consumable
-	PolicyMap        *policymap.PolicyMap
-	L3Policy         *policy.L3Policy
-	L3Maps           L3Maps
-	Opts             *option.BoolOptions // Endpoint bpf options.
-	Status           *EndpointStatus
-	State            string
+	ID                  uint16       // Endpoint ID.
+	Mutex               sync.RWMutex // Protects all variables from this structure below this line
+	DockerContainerName string       // Docker container name.
```

Comment:
Can we call this `ContainerName`? We will rename the `DockerID` to `ContainerID` as well later.